src/main/pages/user-management/organizations.adoc

```diff
@@ -11,49 +11,66 @@ folder: user-management
+The top-level organizations are called root organizations.  Any organization can have zero to a set number of sub-organizations. Only the system admistrator can create root organizations and manage the resources of the root organization. 
 
 [id="creating-an-organization"]
-== Creating an Organization
+== Creating an organization
+
+Only the system administrator is able to create root organizations. The administrator creates sub-organizations. 
+
+To create an organization:
 
-Only the System Admin is able to create root organizations. Organization Admins can create sub-organizations. To create an organization, use the menu in the left sidebar, which leads to the list of organizations. A new page is displayed with all organizations in your system. Click on the top-left button to create a new organization.
+.  Click the menu in the left sidebar.  A new page displays all the organizations in your system.
+
+.  Click on the top-left button to create a new organization.
 
 [id="organization-list"]
-== Organization List
+== List of organizations
 
-The list of all organizations are displayed on organizations page. The list contains the general information for each organization: number of members, total and available RAM and number of sub-organizations.
+A list of all the organizations displays on the organization page. The list contains the following general information for each organization: number of members, total RAM, available RAM, and number of sub-organizations.
 
 [id="adding-organization-members"]
-== Adding Organization Members
+== Adding members to organizations
+
+To add members to organization:
 
-Adding organization members by clicking the "Add" button will display a new popup. You can change an organization member’s role or remove them from the organization at any time.
+. Click *Add* button to add a member.  A new pop-up window displays. 
 
-Note: Users with the green checkmark beside their name already have an account on your Eclipse Chesystem and will be added to the organization. Users without a checkmark do not have an account and will not be added into the organization.
+. Enter member name.  
+
+[NOTE]
+====
+Users with the green checkmark beside their name already have an Eclipse Che account and can be added to the organization. Users without a checkmark do not have an account and cannot be added into the organization.
+====
 
 [id="workspaces-in-organization"]
-== Workspaces in Organization
+== Workspaces in organization
 
-Workspace is created inside of an organization and uses the resources of this very organization. Workspace creator has to choose the organization on workspace creation page.
+You can choose the organizaton of the workspace.  The resources of the organization are used by the workspace. 
```

Comment:
How about:  You can choose which organization that a workstation belongs to on the workspace creation page.  I am not sure if the workspace creator is the sys admin.

Reply:
Two points:

* Your formulation sounds awkward; the orig. was fine IMO.
* My suggestion was wrong because there shjouldn't be 'You' -- instead, it's the org's creator, which could be anyone with rights.

Reply:
How about: A workspace is created inside of an organization and uses the resources of the organization.  The workspace creator chooses the organization on the *Workspace Creation* page.  (I am not sure about using the workspace creator terminology)

Reply:
LGTM